docs/nima/pico/pico_intro.adoc

```diff
@@ -101,16 +175,26 @@ The Pico API provides types that are generally useful at compile time to assign
 
 == Configuration
 
-Pico Config builder
+Configuration can be achieved through a Pico add-on that is based on Helidon's configuration subsystem, and adds support for something called _config-driven services_ using the `@ConfiguredBy` annotation. 
```

Comment:
It is not really "Configuration can be achieved".

Config-driven services is more about the second part of the sentence - where the presence of configuration in the Helidon Config subsystem will be injected directly into a service declared to be configured by that configuration type.